### ISSUE TITLE
mobs regenerate health when stacked

### ISSUE THREAD

**Issue by Kogwork** [Nathat23/StackMob-2#306]
**Describe the bug**
in a fall damage spawner, an mob will die after an hit, mobs stack tho and it seems to reset their health full . also using the new pvp system you can hit a bunch of mobs to kill them in one hit, this plugin removes the abillity to do so.
**To Reproduce**
make a fall damage trap leaving 1 tick of health when hit by a mob, try to slay one and he will fall from a punch, try to slay a stack that fell and the first will die and all the ones after wont.

also in unstacked farmer a long press on the sword will kill all of the mobs, not doable when mobs are stacked
**Expected behavior**
way to register health and store it while the mob is stacked, maybe
by indexing each mob thats added to the stack.

a check for a strong sword attack aginst a stack that each mob has less health than the sword can damage, if thats the case will slay the whole stack when using an strong attack

**Bukkit/Spigot AND StackMob version:**

5.0.2 stackmob
spigot 1.15.2

**Comment by Jirubizu** [comment]
Yeah agreed with this too. Hopefully you could add a way for the mobs to keep their original health
